exporter/collector/metricsexporter.go

```diff
@@ -226,9 +234,27 @@ func (m *metricMapper) resourceMetricsToMonitoredResource(
 	return nil, nil
 }
 
-func instrumentationLibraryToLabels(il pdata.InstrumentationLibrary) labels {
-	// TODO
-	return nil
+func (m *metricMapper) instrumentationLibraryToLabels(il pdata.InstrumentationLibrary) labels {
+	source := m.cfg.MetricConfig.InstrumentationSource
+	if source == "" {
+		source = il.Name()
+	}
+	if source == "" {
+		source = "fallback"
```

Comment:
@aabmass @jsuereth @kjordy 

Should we have a fallback value for this? And what should it be?

Reply:
I think empty string is ok here. If we're going to choose a fallback value it should be something clear like "unspecified" but I don't see much more value between that and just empty string.

Reply:
I'd stick with empty string.  The specification is (now) loose here, there used to be a specififed default but no more (that I can see)

Reply:
I've changed this to use an empty string